proposed/http-handlers/request-handlers.md

```diff
@@ -110,20 +110,20 @@ use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
 
 /**
+ * Participant in processing a server request and response
```

Comment:
Period please. :smile:

Reply:
As I stated in a previous comment (hidden because PR was updated):

The [phpdoc standard disagrees](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#51-summary):

> A Summary MUST end with either
> 
> * a full stop (.) followed by a line break
> * or two sequential line breaks.

Only a single-line summary ends with a period. If a description is included, it does not.